meilisearch-http/src/index_controller/index_actor/actor.rs

```diff
@@ -25,6 +25,7 @@ pub struct IndexActor<S> {
     read_receiver: Option<mpsc::Receiver<IndexMsg>>,
     write_receiver: Option<mpsc::Receiver<IndexMsg>>,
     update_handler: Arc<UpdateHandler>,
+    processing: RwLock<(bool, Option<Uuid>)>,
```

Comment:
can you make this a `RwLock<Option<Uuid>>` instead? when nothing is indexing, it is set to `None`